packages/components/stories/timeline/timeline.stories.js

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+import { boolean, withKnobs } from '@storybook/addon-knobs';
+import { withInfo } from '@storybook/addon-info';
+import Timeline from 'Components/timeline';
+import Theme from '../shared/theme';
+
+const stories = storiesOf('Timeline', module);
+
+stories.addDecorator(withKnobs).addDecorator(withInfo);
+
+stories.add('basic usage', () => (
+    <Theme is_dark={boolean('Theme', true)}>
+        <div style={{ display: 'flex' }}>
+            <div>left component</div>
+            <Timeline>
+                <div
+                    style={{ borderRadius: '4px', border: 'solid 1px #323738' }}
+                    title='Select tokens based on the access you need.'
```

Comment:
Passing title to `div` elements is not considered to be a good choice here, since it is not according to specs. Let's export a component (e.g.`Timeline.Item`) component. 

The usage would be something like: 
```
<Timeline>
    <Timeline.Item title='title'>
         {children_consumer_wants_to_pass_down}
    </Timeline.Item>
</Timeline>
```